### PR TITLE
bugfix: show affiliation btn only if affiliations

### LINKED ISSUE #593: landing page: show affiliation should not show if there's no affiliations
Package version (if known): <version>

## Description

1. Submit a record with one or more authors without affiliations.
2. See on the landing page that the "show affiliations" is displayed despite the authors having no affiliations.
3. If you click "show affiliations" in toggles to "hide affiliations" and you can no longer toggle it back.

Expected that "show affiliations" is not displayed at all.

## Screenshots (if applicable)

Screenshot 2021-02-23 at 00 32 14

### PR DESCRIPTION
**Pull request by mb-wali**
closes #593
inveniosoftware/invenio-rdm-records#304 is still not resolved.

**screenshot**
no-affi

with-aff

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less [hunk 1]
```diff
@@ -153,3 +153,8 @@
          padding-top: 0px !important;
      }
  }
+
+ /* accordion cursor */
+ .ui.accordion .title {
+    cursor: auto !important;
```
Comment (zzacharo) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less:159]: Out of curiosity why is this needed?

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/creators.html [hunk 2]
```diff
@@ -11,10 +12,12 @@
 <div class="ui accordion">
   <div class="title">
     {{ show_creatibutors(record.ui.creators.creators, show_affiliations=True) }}
+    {% if record.ui.creators.affiliations %}
     <a style="cursor: pointer;" class="dropdown">
```
Comment (zzacharo) [inline invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/creators.html:16]: If the below class is needed to make this line work, maybe we should add a class here e.g `affiliation-dropdown` and apply there the rule `cursor: pointer !important; `

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less (hunk 1 above)
Comment (mb-wali) [reply invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less:159]: marked area in the screenshot is the area where the cursor was a pointer.
pointer

**Review by zzacharo — APPROVED** [review]
